src/tests/multihost/ad/test_adparameters_ported.py

```diff
@@ -0,0 +1,1749 @@
+           'cache_credentials': 'True',
+           'krb5_store_password_if_offline': 'True',
+           'fallback_homedir': '/home/%d/%u',
+        }
+        client.sssd_conf(dom_section, sssd_params)
+        # Clear cache and restart SSSD
+        client.clear_sssd_cache()
+
+        # Get uid and gid for the aduser and adgroup
+        get_uid_cmd = f"powershell.exe -inputformat none -noprofile 'Get-ADUser -Identity {aduser} -Properties uidNumber'"
+        cmd = multihost.ad[0].run_command(get_uid_cmd, raiseonerr=False)
+        uid = re.findall("uidNumber.*:[^0-9]+([0-9]+)", cmd.stdout_text)[0]
+        get_gid_cmd = f"powershell.exe -inputformat none -noprofile 'Get-ADGroup -Identity {adgroup} -Properties gidNumber'"
+        cmd = multihost.ad[0].run_command(get_gid_cmd, raiseonerr=False)
+        gid = re.findall("gidNumber.*:[^0-9]+([0-9]+)", cmd.stdout_text)[0]
+
+        # Search for the AD user without uid
+        plain_usr_cmd = multihost.client[0].run_command(
+            f'getent passwd {userplain}',
+            raiseonerr=False
+        )
+        # Search for the group without gid
+        plain_grp_cmd = multihost.client[0].run_command(
+            f'getent group {group_plain}',
+            raiseonerr=False
+        )
+        # Search for the AD user by uid
+        usr_uid_cmd = multihost.client[0].run_command(
+            f'getent passwd {uid}',
+            raiseonerr=False
+        )
+        # Search for the group by gid
+        grp_gid_cmd= multihost.client[0].run_command(
+            f'getent group {gid}',
+            raiseonerr=False
+        )
+        # Search for the user with uid
+        usr_cmd = multihost.client[0].run_command(
+            f'getent passwd {aduser}',
+            raiseonerr=False
+        )
+        # Search for the group with gid
+        grp_cmd = multihost.client[0].run_command(
+            f'getent group {adgroup}',
+            raiseonerr=False
+        )
+        # Run su
+        su_cmd = multihost.client[0].run_command(
+            f'su - {aduser} -c  whoami',
+            raiseonerr=False
+        )
+
+        client.restore_sssd_conf()
+        client.clear_sssd_cache()
+
+        # Evaluate test results
+        assert usr_uid_cmd.returncode == 0, f"User {uid} was not found."
+        assert grp_gid_cmd.returncode == 0, f"Group {gid} was not found!"
+        assert plain_usr_cmd.returncode == 2, f"User {aduser} was found!"
+        assert plain_grp_cmd.returncode == 2, f"Group {adgroup} was found!"
+        assert usr_cmd.returncode == 0, f"User {aduser} was not found."
+        assert grp_cmd.returncode == 0, f"Group {adgroup} was not found!"
+        assert su_cmd.returncode == 0, "The su command failed!"
+
+# TODO: "Account Password Policy Tests"
+# test_0020 "Change user password"
+# test_0021 "User must change password on next logon bz1078840"
+# test_0022 "User account disabled"
+# test_0022 "User account is expired bz1081046"
```

Comment:
FYI, this requires the password policy to be modified on the server for whenever this gets ported.